simplified-tests/src/test/java/org/nypl/simplified/tests/pdf/PdfViewerProviderTest.kt

```diff
@@ -15,30 +15,30 @@ class PdfViewerProviderTest {
   fun supportsPdfBooksWhenEnabled() {
     val preferences = ViewerPreferences(
       flags = mapOf(
-        "enablePDFJSReader" to true
+        "enableOldPDFReader" to true
```

Comment:
I think we need a comment here to explain that setting `enableOldPDFReader`to true _disables_ the _current_ (pdf.js) reader. Otherwise it will be confusing.